docs/index.html

```diff
@@ -0,0 +1,46 @@
+
+    <!DOCTYPE html>
+    <html>
+      <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <meta name="storybook-version" content="3.2.10">
+        <meta content="IE=edge" http-equiv="X-UA-Compatible" />
+        <title>Storybook</title>
+        <style>
+          /*
+            When resizing panels, the drag event breaks if the cursor
+            moves over the iframe. Add the 'dragging' class to the body
+            at drag start and remove it when the drag ends.
+           */
+          .dragging iframe {
+            pointer-events: none;
+          }
+
+          /* Styling the fuzzy search box placeholders */
+          .searchBox::-webkit-input-placeholder { /* Chrome/Opera/Safari */
+            color: #ddd;
+            font-size: 16px;
+          }
+
+          .searchBox::-moz-placeholder { /* Firefox 19+ */
+            color: #ddd;
+            font-size: 16px;
+          }
+
+          .searchBox:focus{
+            border-color: #EEE !important;
+          }
+
+          .btn:hover{
+            background-color: #eee
+          }
+        </style>
+        
+      </head>
+      <body style="margin: 0;">
+        <div id="root"></div>
+        <script src="static/manager.13ba8ab67269fda303a8.bundle.js"></script>
```

Comment:
Is this supposed to be hard-coded like this? Have you looked into using React Storybook with Create React App? Would make you life a lot easier building the actual demo app for the components.

I don't think we should be checking in the bundles...

Reply:
This is generated by React Storybook.
I will look into ways of having Travis update the docs for us, or maybe having a different release process that includes that.